SUPPORT.md

```diff
@@ -0,0 +1,48 @@
+# Getting support for Julia
+
+As well as here on GitHub, Julia is supported by several other online and offline communities.
+If you have come here to open an issue on the GitHub repository, you might want to want to consider the following options first.
```

Comment:
I mean, if you have an issue, you should post it rather than go somewhere else. Perhaps clarify to something like "if you have come here for help" or "if you are not sure you have found a bug" etc, as formulated in https://github.com/JuliaLang/julia/blob/master/.github/ISSUE_TEMPLATE.md